.golangci.yml

```diff
@@ -47,6 +47,9 @@ issues:
   exclude:
     - 'declaration of "(err|ctx)" shadows declaration at'
   exclude-rules:
+    - path: server/standup/main.go
```

Comment:
why we have excluded this file?

Reply:
Govet was trying to space optimize a struct which was provided by a package and hence was immutable
Excluded the file after discussion with manoj